w02h03/test/pgdp/UnitTests.java

```diff
@@ -0,0 +1,44 @@
+package pgdp;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import pgdp.math.PinguSqrt;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Random;
+
+public class UnitTests {
+
+    private static final ByteArrayOutputStream out = new ByteArrayOutputStream();
+    private static final PrintStream stdout = System.out;
+
+    @BeforeAll
+    public static void prep()
+    {
+        System.setOut(new PrintStream(out));
+    }
+
+    @AfterAll
+    public static void clean()
+    {
+        System.setOut(stdout);
+    }
+
+    @Test
+    public void checkSqrt() {
```

Comment:
```suggestion
    void checkSqrt() {
```
Tests sind package-private (vgl. wenn Intellij einen Test-Kopf autogeneriert)